content/doc/book/managing/nodes.adoc

```diff
@@ -14,6 +14,60 @@ endif::[]
 
 = Managing Nodes
 
+== Launch Java Web Start agent via Windows Scheduler
+
+If you are having trouble getting the Java Web Start agent installed as a Windows service (i.e., you followed https://wiki.jenkins.io/display/JENKINS/Installing+Jenkins+as+a+Windows+service[the instructions on installing the agent as a service here] but it didn't work), an alternative method of starting the service automatically when Windows starts is to use the Windows Scheduler. 
+
+We take advantage of the Windows Scheduler's ability to run command at system startup
+
+. Configure your node to use the "Launch agents via Java Web Start" launch method
+* Click Save
+. Note the command required to launch the agent
+* On the new agent node's Jenkins page, note the agent command line shown. 
+** It will be like: java -jar slave.jar -jnlpUrl http://<JenkinsHostName>:8080/computer/<nodeName>/slave-agent.jnlp -secret <some_long_hex_string>
```

Comment:
```suggestion
** It will be like: java -jar agent.jar -jnlpUrl http://<JenkinsHostName>:8080/computer/<nodeName>/slave-agent.jnlp -secret <some_long_hex_string>
```